plugins/azure/kubernetesservice/rbacEnabled.spec.js

```diff
@@ -0,0 +1,99 @@
+var expect = require('chai').expect;
+var roleBasedAccessControl = require('./rbacEnabled');
+
+const managedClusters = [
+    {
+        id: '/subscriptions/123/resourceGroups/aqua-resource-group/providers/Microsoft.ContainerService/managedClusters/tes-cluster',
+        location: 'eastus',
+        name: 'tes-cluster',
+        type: 'Microsoft.ContainerService/ManagedClusters',
+        provisioningState: 'Succeeded',
+        kubernetesVersion: '1.18.14',
+        dnsPrefix: 'tes-cluster-dns',
+        fqdn: 'tes-cluster-dns-f7b98b1e.hcp.eastus.azmk8s.io',
+        enableRBAC: true,
+        maxAgentPools: 10,
+    },
+    {
+        id: '/subscriptions/123/resourceGroups/aqua-resource-group/providers/Microsoft.ContainerService/managedClusters/tes-cluster',
+        location: 'eastus',
+        name: 'tes-cluster',
+        type: 'Microsoft.ContainerService/ManagedClusters',
+        provisioningState: 'Succeeded',
+        kubernetesVersion: '1.18.14',
+        dnsPrefix: 'tes-cluster-dns',
+        fqdn: 'tes-cluster-dns-f7b98b1e.hcp.eastus.azmk8s.io',
+        enableRBAC: false,
+        maxAgentPools: 10,
+    }
+];
```

Comment:
Use double quotes notation to define objects
#586

Reply:
Done as suggested.
@AkhtarAmir